java/code/src/com/redhat/rhn/frontend/action/channel/manage/ChannelPackageMenuAction.java

```diff
@@ -49,7 +50,7 @@ public ActionForward execute(ActionMapping mapping,
         long cid = requestContext.getRequiredParam("cid");
 
         Channel chan = ChannelFactory.lookupByIdAndUser(cid, user);
-        if (!user.hasRole(RoleFactory.CHANNEL_ADMIN) &&
+        if (!user.isMemberOf(AccessGroupFactory.CHANNEL_ADMIN) &&
                 !UserManager.verifyChannelAdmin(user, chan)) {
             throw new PermissionCheckFailureException(RoleFactory.CHANNEL_ADMIN);
```

Comment:
we have a few exceptions there are thrown and user the old roles. This is more a nitpick and can be solved in a follow-up PR after Beta2.